requirements-crypto-test.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements-crypto.txt
+requests==2.23.0
```

Comment:
Why not use the dev requirements for the test env?

Reply:
Because #387 . After that is fixed, I've prepared a rebased [v2](https://github.com/vamposdecampos/covid-19-ro-help/tree/crypto-test-v2) branch that could be used instead.

Reply:
Ugh, yeah. 🤕 I will try to find the time to backport the dependency management from https://github.com/code4romania/votong and get this fixed.